internal/usage/usage.go

```diff
@@ -154,6 +154,7 @@ const (
 	SubscriptionID      = "Unique identifier of the Azure subscription in which the VNet resides."
 	GCPProjectID        = "Unique identifier of the GCP project in which the network peer resides."
 	Network             = "Unique identifier of the Network Peering connection in the Atlas project."
+	IntegrationRegion   = "Indicates which API URL to use, either US or EU. The integration service will use US by default."
```

Comment:
[non-blocking] Data dog has this region as well, if they are the same good but just a heads up that this may have to change later for data dog or feel free to prepare this now making this specific, ie: `OpsGenieRegion`